.github/workflows/php-library-static.yml

```diff
@@ -7,10 +7,14 @@ on:
         required: false
         default: true
         type: boolean
+      php-version:
+        required: false
+        type: string
+        default: '8.1'
   
 jobs:
   phpstan:
-    name: PHPStan 
+    name: PHPStan - ${{ inputs.php-version }}
```

Comment:
It'd be good to have the stability in all the job names too (`prefer-stable`/`prefer-lowest`) to avoid duplicate job names.